InformationScripting/src/queries/AstQuery.cpp

```diff
@@ -350,6 +347,50 @@ TupleSet AstQuery::typeFilter(TupleSet input)
 	return result;
 }
 
+TupleSet AstQuery::attribute(TupleSet input)
+{
+	TupleSet result;
```

Comment:
Move this right before the `if(scope()...`.

In general, definitions should also be immediately after first use.